libmozdata/config.py

```diff
@@ -11,12 +11,21 @@
 
 
 class Config(object):
-    def get(section, option, default=None):
-        raise NotImplementedError
+    def __init__(self):
+        self.local_config = {}
+
+    def set(self, section, option, value):
+        self.local_config[(section, option)] = value
```

Comment:
Since this adds a fallback instead of overwriting the current value, the name `set()` could be a bit confusing. It would be clearer to call it something like `set_default()`.